features/shopcarts.feature

```diff
@@ -111,4 +111,22 @@ Scenario: List all shopcarts
     And I should see "1.99" in the results in "Shopcart" page
     And I should not see "Lillisluo" in the results in "Shopcart" page
     And I should not see "Pear" in the results in "Shopcart" page
-    And I should not see "199.99" in the results in "Shopcart" page
\ No newline at end of file
+    And I should not see "199.99" in the results in "Shopcart" page
+
+Scenario: List all items under a shopcart
+    When I visit the "Shopcart" api page
+    And I set the "Name" to "Wan-Yu" in "Shopcart" page
+    And I press the "Search" button in "Shopcart" page
+    Then I should see the message "Success"
+    When I copy the "Id" field in "Shopcart" page
+    And I visit the "Item" api page
+    And I set the "Shopcart ID" within clipboard in "Item" page
+    And I press the "Search" button in "Item" page
+    Then I should see the message "Success"
+    And I should see "Apple" in the results in "Item" page
+    And I should see "1" in the results in "Item" page
```

Comment:
Given it's the list shopcart items API, I guess we only need to check the item name column in the feature file? Because take the quantity column for example, we're unable to distinguish which item `1` is referring to? (e.g. it could be Apple or Orange or Mango)

Reply:
Yes you are correct!